derrida/books/management/commands/instance_data.py

```diff
@@ -24,27 +24,31 @@ class Command(reference_data.Command):
 
     #: fields for CSV output
     csv_fields = [
-        'id', 'item_type', 'title', 'short_title',
+        'id', 'item_type', 'title',
         'alternate_title', 'work_year', 'copyright_year',
         'print_date', 
         'authors', 'contributors', 'publisher', 'pub_place',
         'is_extant', 'is_annotated', 'is_translation', 'has_dedication',
-        'has_insertions', 'copy', 'dimensions', 'work_uri',
+        'has_insertions', 'copy', 'dimensions',
         'subjects', 'languages', 'journal_title',
         'collected_work_title', 'collected_work_uri',
         'start_page', 'end_page',
         'has_digital_edition', 'catalog_uri', 'zotero_id'
     ]
 
     def handle(self, *args, **kwargs):
-        base_filename = 'instances'
+        base_filename = 'library'
         if kwargs['directory']:
             base_filename = os.path.join(kwargs['directory'], base_filename)
 
         filtered_instances = Instance.objects.filter(Q(cited_in__isnull=False) | 
                 Q(reference__isnull=False) |
                 Q(collected_set__cited_in__isnull=False) |
-                Q(collected_set__reference__isnull=False)) \
+                Q(collected_set__reference__isnull=False) |
+                # HACK: Derrida team added annotations and insertion data to
+                #  these three instances.
+                # This is fine, right? Totally fine. Nothing to see here.
+                Q(pk__in=[243, 247, 177])) \
```

Comment:
243 is `de la grammatologie`! we shouldn't include that one for sure; I'll update insertions export to exclude 

247 is a B copy; I bet the other copy has references

177 seems to have references https://derridas-margins.princeton.edu/library/rousseau-oeuvres-completes-tome-i-1959/references/ — my quick check indicated this query includes it

Reply:
I'm ok with hard-coding this in this instance because it doesn't seem like an easy way to query for this, we don't have a data team to check & correct the errors, and we can live with the brittleness for the time needed to run this

Reply:
✅ 57682cb5ffb934fa2d78ada79af7ccc17e6e3d8b removed de la grammatologie